pkg/domain/podman/component.go

```diff
@@ -0,0 +1,73 @@
+package podman
+
+import (
+	"strconv"
+
+	"github.com/skupperproject/skupper/client/podman"
+	"github.com/skupperproject/skupper/pkg/domain"
+)
+
+type SkupperComponentHandlerPodman struct {
+	cli *podman.PodmanRestClient
+}
+
+func NewSkupperComponentHandlerPodman(cli *podman.PodmanRestClient) *SkupperComponentHandlerPodman {
```

Comment:
Related to the abstraction pattern topic noted prior, might it be better to not include 'Podman' in func naming?